### ISSUE TITLE
adapt-contrib-hotgraphic: use of role=list breaks accessibility

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2450]
### Subject of the issue/enhancement/features
The addition of `role=list` in v4.2.1 prevents Jaws users from being able to access the hotspots

From what I can figure out it's because there's no text on the hotspot pins 

When I changed the `<button>` element to this:
```handlebars
<div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}">this is hotspot item {{@index}}</div>
```
Jaws let me access all the elements and announced them in the expected manner.

(I'm not sure the change from `<button>` to `<div>` was necessary, that just happened to be how I had it set up and I ran out of time for further testing)

I suspect this `role` attribute may need to be dropped.

### Your environment
* FW 4.2.0/Hot graphic 4.2.1
* IE11/jaws 2018

**Comment by moloko** [comment]
although https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/176 does fix this, we've had the following feedback from our client's a11y tester which seemed - to me anyway - pretty reasonable:
> The buttons are not announced as buttons – there is no indication for screen reader users that they are actionable items. Even though there is a brief instruction at the beginning of each section (“select each item to get more information”), it is very unlikely that users will try to active content that it is announced as static text by screen readers. To fix this, `role=“listitem”` must be removed from the source code of each button, as this overrides the role of button; HTML list mark-up must be used in place of ARIA to indicate that the buttons are part of a list  (`<ul>` and `<li>`). 

I've had a quick look at a few 'best practice' examples on the WAI site (like [this one](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html))and they do seem to favour using `<ul>` and `<li>` for UI design patterns similar to this.

**Comment by oliverfoster** [comment]
lovely, i shall hupdate.

**Comment by brian-learningpool** [comment]
@oliverfoster, just checking if this work mentioned on the previous two comments is still outstanding after v4.2.2?

**Comment by moloko** [comment]
@brian-learningpool  should all be done & dusted now (as of v4.2.2)
